ctc-oss/omega-edit
# Java Bindings

Easiest way to test out the API is:
- download the `omega-edit-java` zip
- extract and cd into the folder
- run `./bin/omega-edit`.
- browse to `http://localhost:9000/omega_license_get` in the browser and you should get the license text
- For Mac:
  - You need to go to System Preferences >> Security and Privacy >> General and Allow Access for libomega_edit.dylib
    - This may not show up till you run the server
  - After access has been given when you try to hit the API you will get a prompt to `Open` the `libomega_edit.dylib` file
